plugins/CoreHome/javascripts/popover.js

```diff
@@ -107,6 +107,10 @@ var Piwik_Popover = (function () {
         if (container !== false) {
             $('.ui-dialog').css({margin: '0 0'});
             container.dialog("option", "position", {my: 'center', at: 'center', of: '.ui-widget-overlay', collision: 'fit'});
+            // in some cases jQuery UI fails to place the dialog correctly and set the top values to something negative
```

Comment:
From what my debugger told me there is some rounding error going on when placing the dialog: https://github.com/jquery/jquery-ui/blob/0df6e658307f8936a477deb9674d643d18a2469b/ui/position.js#L381-L382

The ones consistently broken showed rounding errors for `overTop` and/or `overBottom`, e.g. `overBottom == 776.6399999999999` and `overTop == 776.64`. That triggers the wrong position branch and breaks :shrug:

Reply:
Nice find @mneudert. Should we perhaps compare only values converted to int?

Reply:
But I see it'd be within the UI lib itself, not in our control here.